lib/matplotlib/_api/__init__.py

```diff
@@ -120,6 +120,8 @@ def check_in_list(_values, *, _print_supported_values=True, **kwargs):
     --------
     >>> _api.check_in_list(["foo", "bar"], arg=arg, other_arg=other_arg)
     """
+    if not kwargs:
+        raise ValueError("No argument to check!")
```

Comment:
In general, signature mismatches should be TypeErrors.

Reply:
Fixed!